Lib/test/test_typing.py

```diff
@@ -7749,6 +7750,7 @@ class CustomerModel(Decorated, frozen=True):
                 "eq_default": True,
                 "order_default": True,
                 "kw_only_default": False,
+                "frozen_default": False,
```

Comment:
I think `frozen_default` should be `True` here.
Because we define `class CustomerModel(Decorated, frozen=True):`, we should pass `frozen_default=True` to `dataclass_transform` in this specific case.

Reply:
PEP 681 says:

> At runtime, the `dataclass_transform` decorator’s only effect is to set an attribute named `__dataclass_transform__` on the decorated function or class to support introspection. The value of the attribute should be a dict mapping the names of the `dataclass_transform` parameters to their values.

I would therefore expect the `Decorated` type to have a `__dataclass_transform__` attribute, but not `CustomerModel`, and therefore I would not expect the `frozen=True` on `CustomerModel` to have any effect on `__dataclass_transform__`.

Reply:
Oh, I see. If we want to verify that `frozen_default` is `True` in `__dataclass_transform__` when it should be, how about if I set `frozen_default=True` in `test_metaclass`? It's the only test where `frozen` isn't used. We could also add a test specifically to test `__dataclass_transform__`.